agentMET4FOF/agents.py

```diff
@@ -291,7 +291,7 @@ def buffer_store(self, agent_from: str, data=None):
 
```

Comment:
We should document the new parameter `concat_axis` in the docstring right away. Same applies to `def _concatenate(self, iterable, data, concat_axis=0)` in line 681 and `def store(self, agent_from, data=None, concat_axis=0):` in line 785.